Bottom: d955eaf6b5c14e292de7c2e88d766ffe3dcf1c4c
Top:    6d7fa97c947bb5efd96d1fff321e7b01b9381f77
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 21:08:36 +0000

Refresh of well-on-the-way-to-completely

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index bfd0875..dc9c242 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -465,6 +465,7 @@ object3d_from_board_outline (void)
 #endif
           edge_info_set_stitch (UNDIR_DATA (edges[2 * npoints + i]));
         }
+
     }
 
     if (1) {
@@ -518,9 +519,6 @@ object3d_from_board_outline (void)
       face3d_set_cylindrical (cylinder_faces[0], COORD_TO_STEP_X (PCB, MM_TO_COORD (45.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 0., /* A point on the axis of the cylinder */
                                         0., 0., 1.,             /* Direction of the cylindrical axis */
                                         5.);                   /* Radius of cylinder */
-      face3d_set_cylindrical (cylinder_faces[0], COORD_TO_STEP_X (PCB, MM_TO_COORD (45.)), COORD_TO_STEP_Y (PCB, MM_TO_COORD (45.)), 0., /* A point on the axis of the cylinder */
-                                        0., 0., 1.,             /* Direction of the cylindrical axis */
-                                        5.);                   /* Radius of cylinder */
       face3d_set_normal (cylinder_faces[0], 1., 0., 0.);       /* A normal to the axis direction */
                                    /* XXX: ^^^ Could line this up with the direction to the vertex in the corresponding circle edge */
       object3d_add_face (board_object, cylinder_faces[0]);
